database/migrations/20170531173912_organizations.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = knex =>
+  knex.schema.createTable( 'organizations', table => {
+    table.increments( 'id' ).primary()
+    table.integer( 'user_id' ).notNullable()
```

Comment:
Is this a foreign key? If so should also use the `table.foreign(...).references(...)` syntax. See other migrations for examples.